pkgs/tools/networking/nordvpn/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, lib, fetchurl, openvpn, libxml2, autoPatchelfHook, dpkg }:
+
+let
+  version = "3.10.0-1";
+in stdenv.mkDerivation {
+  pname = "nordvpn";
+  inherit version;
```

Comment:
```suggestion
stdenv.mkDerivation {
  pname = "nordvpn";
  version = "3.10.0-1";
```

Reply:
Curious for this one, my understanding for a lot of the pkgs in nixpkgs, is to use reuse version in fetching + the derivation itself

Reply:
Since I ain't using `rec` here (my understanding is to avoid using `rec` if possible, and use `let .. in`)

Reply:
You can safely use rec in mkDerivation.

Reply:
Ah k